src/routes/api/installation/api-installation-router.ts

```diff
@@ -7,25 +7,27 @@ import { ApiInstallationSyncPost } from "./api-installation-sync-post";
 import { ApiInstallationGet } from "./api-installation-get";
 
 export const ApiInstallationRouter = Router({ mergeParams: true });
+const subRouter = Router({ mergeParams: true });
+ApiInstallationRouter.use(`(/githubapp/:gitHubAppId(\\d+))?`, subRouter);
 
-ApiInstallationRouter.post(
+subRouter.post(
```

Comment:
Curious what would be the path to trigger resync for a cloud customer?

Reply:
Seems the resync is routed in the `ApiRouter` as
```javascript
ApiRouter.post(`/:uuid(${UUID_REGEX})?/resync`,..., async (req, res) => {});
```

So it is not included inside this ApiInstallationRouter.